### PR TITLE
Portfolio: mobile version (Day4 task)

### PR DESCRIPTION
**Pull request by franciscoPonceDev**
- Add the Recent works section.

- Add the about me section.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -75,3 +85,261 @@ body {
...
+/* I've separated the about me in two sections in order to use the background image properly */
+
+/* About me 1st section */
+
+.about-text {
+  background-image: url(./assets/About-me_background_image.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: bottom;
+}
```
Comment (Hamzaoutdoors) [inline styles.css]: - [x]  Great job adding this background image as `background_image` and not `img`, but unfortunately this background is sticking a little bit to the left. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make this background image closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) 😸 .

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
  |

File: styles.css [hunk 2]
```diff
@@ -75,3 +85,261 @@ body {
...
+  background-color: #fff5e1;
+  border: 1px solid #f55b00;
+  color: #f55b00;
+}
+
+button:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
```
Comment (Hamzaoutdoors) [inline styles.css:174]: **Nice job** for making this button look the same as in the Figma design.  by adding the **Active**, **Hover**, **Pressed** button styling effects. buttons are a vital element that creates a positive and productive user experience.

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hi @franciscoPonceDev , I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
( - **N.A.** )

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (franciscoPonceDev) [reply styles.css:174]: Hello @Hamzaoutdoors!
I'm sorry. I didn't understand what exactly are the requested changes here.
As you can see on the conde I've already added the :hover and :active selectors. And when I hover the mouse through the buttons, they change the background-color, border, and color. The same when I press them, they are the same as the Figma template.

Comment (Hamzaoutdoors) [reply styles.css:174]: Hello @franciscoPonceDev
 - In this comment, I am just **highlighting** your efforts working on that 👍🏼 👍🏼, sorry for making you confused in this part

Comment (franciscoPonceDev) [reply styles.css:174]: Oh Thank you!! It is great to receive positive feedback from you! :)

**Review by HENRYKC24 — APPROVED** [review]
Hi @franciscoPonceDev :wave:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**